addons/calendar_sms/tests/test_calendar_sms.py

```diff
@@ -13,7 +13,7 @@ class TestCalendarSms(TransactionCase, SMSCase):
     def setUpClass(cls):
         super(TestCalendarSms, cls).setUpClass()
 
-        now = fields.datetime.now()
+        now = cls.env.cr.now()
```

Comment:
That fix feel strange. We should be able to use the time without milliseconds here.